infrastructure/ci_iam.tf

```
@@ -0,0 +1,47 @@
+locals {
+  ci_role_name = trimprefix(data.aws_arn.ci_role.resource, "role/")
+}
+
+data "aws_arn" "ci_role" {
+  arn = local.ci_role_arn
+}
+
+resource "aws_iam_role_policy" "ci_s3_publish" {
+  name   = "content-s3-publish"
```

```suggestion
  name   = "ci-s3-publish"
```

or just omit the name entirely, I don't think we usually set it do we?